tools/fetch.py

```diff
@@ -98,15 +99,15 @@ def vx_start_external(nodelist,outdir,cmd, timeout=0, threadcount=20):
     args = build_vx_args_external(cmd)
     vxargs.start(None, threadcount, nodelist, outdir, False, args, timeout)
 
-def build_vx_args(shell_cmd):
+def build_vx_args(shell_cmd, user):
```

Comment:
Can we do user=site_admin here, since I think this is the user that will be used in the vast majority of cases?